Sming/Makefile-rboot.mk

```diff
@@ -173,31 +179,7 @@ endif
 
 EXTRA_INCDIR += $(SMING_HOME)/include $(SMING_HOME)/ $(LWIP_INCDIR) $(SMING_HOME)/system/include $(SMING_HOME)/Wiring $(SMING_HOME)/Libraries $(SMING_HOME)/SmingCore $(SMING_HOME)/Services/SpifFS $(SDK_BASE)/../include $(THIRD_PARTY_DIR)/rboot $(THIRD_PARTY_DIR)/rboot/appcode $(THIRD_PARTY_DIR)/spiffs/src
 
-USER_LIBDIR  = $(SMING_HOME)/compiler/lib/
```

Comment:
@gbl08ma Please, make only one logical change per PR. It makes things easier for us to review and maintain the code. If there are other issues in the makefile(s) then make a second, third, etc PR. Therefore I would like to ask you to remove all changes that are not related to "Fix debug messages when rBoot is enabled". If you cannot do it then I will have to rebase this PR later this week.